### DISCUSSION TITLE
Italian translation

### DISCUSSION THREAD

**Discussion by pdo59** [Part-DB/Part-DB-server#368] [General]
The Italian translation is missing in the PartDB, if you provide me with the instructions I am available to do so.

**Comment by jbtronics** [comment]
The easiest way to work on Part-DB translations is to use CrowdIn. I added Italian as language on the crowdin project page: https://part-db.crowdin.com/part-db/it

The source strings are given as some translation keys, which might not directly show what the meaning of the translation should be.
Therefore you should expand the "Other languages" panel in the translation editor, to see the english version of the translation:

<img width="1916" alt="image" src="https://github.com/Part-DB/Part-DB-server/assets/5410681/1dd73def-68f6-4e1c-8043-66369ea26b3f">

**Reply by pdo59** [reply]
Ok, I'll try

**Comment by pdo59** [comment]
Hi, I think the translation is complete.  I would like to take a walk around the app to make sure everything is congruent; How can I do?

**Reply by jbtronics** [reply]
Thank you for that.

I have merged the translations into the master branch.
You can view them in the demo under http://part-db.herokuapp.com/it/

When they are finished, I add italian to the language selector dropdown menu in the navbar.

**Comment by pdo59** [comment]
Ok, I did some checks but, in the demo, if I log in as a user, the Italian language is missing. Anyway I think my work is done, I will use PartDB in my server and fix any errors if necessary.
What are the steps to make Italian the default language in my installation?

**Comment by jbtronics** [comment]
The translations are now officially available in Part-DB 1.7.2 and you can select them via the language selector in the navbar.

To make italian the global default language, set `DEFAULT_LANG=it` in your `env.local` file, or your docker env settings.

**Comment by pdo59** [comment]
Thanks! 
If I find other errors in the Italian translation I will continue to use crowdin to correct them right? When you add terms to translate with the new versions, will I still find them in crowdin?

**Reply by jbtronics** [reply]
Yes. Crowding gets automatically synced with the source code.

**Reply by Efois** [reply]
Could I still help in this task?

**Reply by jbtronics** [reply]
Login at crowdin, there you can edit the translations: https://part-db.crowdin.com/

I think most of the strings are already translated to italian, but you can still proofread.

**Comment by pdo59** [comment]
Ok, I will make sure to keep the translation updated.  Your work is very beautiful, I hope that many Italian hobbyists use it.
